### PR TITLE
Update getting started

### PR DESCRIPTION
**Pull request by shahzebsiddiqui**
This PR is meant to address documentation mostly Getting Started page which would need some more sections. The docs would need a lot more work.

I didn't run the examples using ``docgen`` script since it's not working due to BuildTestCommand in #287   that is used to get the output of the commands.

### CODE REVIEW

File: buildtest/buildsystem/base.py [hunk 1]
```diff
@@ -1,7 +1,4 @@
 """
-BuildspecParser: loader and manager for Buildspec, and schema validation
-Copyright (C) 2020 Vanessa Sochat.
```
Comment (vsoch) [inline buildtest/buildsystem/base.py:3]: If we are using the LICENSE in the root of the repo in favor of in the header, then just make sure to do a grep for Copyright to ensure that all are removed. I agree this is much easier to maintain in one spot!

File: docs/index.rst [hunk 2]
```diff
@@ -2,11 +2,9 @@ buildtest
 ==========
 
 `buildtest <https://github.com/buildtesters/buildtest>`_  is
-a testing framework designed for HPC Software Stack Testing that is compatible with Lmod module system.
-buildtest provides a set of YAML keys to write test configuration (YAML) that buildtest translates into complex test
-scripts. This allows users to focus on writing test configuration with minimal knowledge of the
-underlying system. Test configuration are reusable between HPC sites with the goal of sharing tests between the
-HPC community.
+a framework designed to automate testing of HPC systems by writing tests in YAML markup language.
```
Comment (vsoch) [inline docs/index.rst]: Maybe say "testing of HPC and other systems?" It's not just for HPC.

**Review by vsoch — APPROVED** [review]
After you confirm that the grep looks okay, this is good to merge!

**Comment by shahzebsiddiqui** [comment]
> After you confirm that the grep looks okay, this is good to merge!

Yep, i checked and removed all Copyrights, now they are present only in ``LICENSE`` and ``docs/conf.py``. 



```
(buildtest) ssi29@ag-mxg-hulk090> grep -iR "Copyright" .
./docs/conf.py:copyright = "2017-2020, Shahzeb Siddiqui and Vanessa Sochat"
Binary file ./docs/slides/buildtest_hpckp17.pdf matches
./LICENSE:Copyright (c) 2017-2020, Shahzeb Siddiqui and Vanessa Sochat
./LICENSE:The above copyright notice and this permission notice shall be included in all
./LICENSE:AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
./.git/logs/refs/heads/update_getting_started:b9987e18d694b2ea6c817a2a3d7791c6794394a1 08ad0dac72ff89842bfe8bb6dd82ac791b21f0d9 Shahzeb Siddiqui <shahzebmsiddiqui@gmail.com> 1587755272 -0400      commit: remove copyright from test files, this is captured in LICENSE.
./.git/logs/HEAD:b9987e18d694b2ea6c817a2a3d7791c6794394a1 08ad0dac72ff89842bfe8bb6dd82ac791b21f0d9 Shahzeb Siddiqui <shahzebmsiddiqui@gmail.com> 1587755272 -0400 commit: remove copyright from test files, this is captured in LICENSE.
./.git/hooks/pre-rebase.sample:# Copyright (c) 2006, 2008 Junio C Hamano
./.git/COMMIT_EDITMSG:remove copyright from test files, this is captured in LICENSE.

```

The only other location is ``setup.py`` , it contains ``author`` field for pypi package. I didn't make this change yet since we haven't released the package yet.

**Review by vsoch — APPROVED** [review]

**Pull request merged**